apps/web/components/eventtype/EventSetupTab.tsx

```diff
@@ -301,7 +302,11 @@ export const EventSetupTab = (
                     <div className="flex items-center">
                       <img
                         src={eventLocationType.iconUrl}
-                        className="h-4 w-4 dark:invert-[.65]"
+                        className={classNames(
+                          "h-4 w-4",
+                          !eventLocationType.iconUrl?.startsWith("/app-store") && "dark:invert",
+                          eventLocationType.iconUrl?.includes("-dark") && "dark:invert"
```

Comment:
maybe worth putting into a helper function since we do this to every "locationIcon"

Reply:
I really don't want an /app-store check here. That's very fragile.

Do we not follow the same -dark approach for app-store svgs?

Reply:
Yeah duplicated logic like this is more fragile. We should consolidate it

Reply:
the /app-store check was used before. essentially this excludes the hard coded .svgs which we are inverting (map.svg and link.svg)